rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/core/RMCore.java

```diff
@@ -1227,18 +1259,32 @@ public BooleanWrapper nodeIsAvailable(String nodeUrl) {
 
             if (node == null) {
                 logger.warn("Cannot set node as available, the node is unknown: " + nodeUrl);
+                if (logger.isDebugEnabled()) {
+                    logger.warn("Known nodes are: " + Arrays.toString(allNodes.keySet().toArray()));
```

Comment:
logger.debug seems more appropriate here, warning is handled in the previous log